vnext/src/Libraries/Components/Button.windows.js

```diff
@@ -20,10 +20,15 @@ const TouchableHighlight = require('./Touchable/TouchableHighlight');
 // Windows]
 const View = require('./View/View');
 const invariant = require('invariant');
+// [Windows
+const ReactNative = require('react-native');
```

Comment:
Since this change is within react-native itself, you can do a relative path, like how `View` is imported.